src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -91,8 +92,6 @@ protected virtual Task Stop()
 
     async Task CreateSeedData(ICreateSeedData instance)
     {
-        Log.Info("Creating or purging queues...");
-        await CreateOrPurgeAndDrainQueues().ConfigureAwait(false);
         Log.Info("Creating send only endpoint...");
```

Comment:
Well, even better is replacing this with the [horizontal ellipsis character (…)](http://www.fileformat.info/info/unicode/char/2026/index.htm).

What do you think?